.circleci/config.yml

```diff
@@ -65,7 +65,7 @@ executors:
     docker:
       - image: *latest-android
     environment:
-      GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"'
+      GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx3g -Dorg.gradle.daemon=false -Dfile.encoding=utf-8 -XX:+HeapDumpOnOutOfMemoryError"'
```

Comment:
I ssh-ed to circle ci and tried different settings but could not quite get all android to build all the time before my session expired.